### ISSUE TITLE
Wrong typescript definition of Surveys.getAvailableSurveys

### ISSUE THREAD

**Issue by tedcurrent** [Instabug/Instabug-React-Native#308]
Hey

In index.d.ts:
`function getAvailableSurveys(availableSurveysCallback: () => void): void;`

it should likely be something like this (not really sure about the `Survey` type)
```
function getAvailableSurveys(availableSurveysCallback: (surveys: Survey[]) => void): void;

interface Survey {
  title: string
}
```

**Comment by salmamali** [comment]
@tedcurrent  Thanks for pointing this out. We'll add this change to the next release.

**Comment by salmamali** [comment]
@tedcurrent fixed in version 8.4.1
